examples/ensemble/plot_gradient_boosting_categorical.py

```diff
@@ -0,0 +1,93 @@
+"""
+========================================
+Categorical Support in Gradient Boosting
+========================================
+
+.. currentmodule:: sklearn
+
+This example, we will compare the performance of
+:class:`~ensemble.HistGradientBoostingRegressor` using one hot encoding
+and with native categorical support.
+
+We will work with the Ames Lowa Housing dataset which consists of numerical
+and categorical features, where the houses' sales prices is the target.
```

Comment:
Are the categorical features useful for this classification task? It may be worth it to add another example where the categorical features are dropped, training should be faster but predictive performance should be worse. Dropping categorical features is another way to deal with them (in a dummy way).

Reply:
For this dataset, the categories do not matter as much. So I will be on the lookout for a nicer dataset.